how-to/networking/dnssec-troubleshooting.md

```diff
@@ -0,0 +1,260 @@
+(dnssec-troubleshooting)=
+# Basic DNSSEC troubleshooting
+
+Some of the troubleshooting tips that will be shown here are focused on the bind9 DNS server and its tools, but the general principle applies to DNSSEC in all implementations.
```

Comment:
Should bind9 be BIND9? (If so, might need to do a global search/replace)

Reply:
I'm not sure, sometimes I mean the package. The upstream name, though, is capitalized indeed: BIND (without the 9). There used to be a bind8, and also bind4 iirc. Bind8 lasted a long time, and when 9 came, it was big news.

Reply:
Looking at other docs we have, outside this PR, looks like we have been capitalizing it indeed, when referring to the project, and not the package.

Reply:
done globally